hironx_moveit_config/config/HiroNX.srdf

```diff
@@ -49,14 +49,24 @@
         <link name="LHAND_JOINT2_Link" />
         <link name="LHAND_JOINT3_Link" />
     </group>
+    <group name="both_arm">
+        <group name="right_arm" />
+        <group name="left_arm" />
+    </group>
     <!--GROUP STATES: Purpose: Define a named state for a particular group, in terms of joint values. This is useful to define states like 'folded arms'-->
-    <group_state name="home" group="left_arm">
```

Comment:
Yes.
- This particular `group_state` setting only with left_arm isn't found used in anywhere within rtmros_hironx package. And I think it hasn't been useful to anybody.
- `left_arm` is already used as a name of move group as well, so it can be a bit confusing.

And we can use the new `init_rtm`, much like `goInitial()` from hrpsys interface. [Test for it is added](https://github.com/start-jsk/rtmros_hironx/pull/389/files#diff-2af18f98392d623abf5fe4cd664f5420R146).